### PR TITLE
Changed alpha for convergence and added swarming example

### PR DESCRIPTION
**Pull request by EnricaSo**
This pull request adds an example to the matlab folder. The cost is a Nonlinear LS function. Convergence only happens if alpha is changed from 1.0 to 0.2. 
Alpha can be found here: https://github.com/acados/acados/blob/master/acados/ocp_nlp/ocp_nlp_sqp.c#L206

### CODE REVIEW

**Comment by giaf** [comment]
Hi Enrica!

Thanks for committing the swarm example. Now it is possible to set the SQP `step_length` directly from the matlab interface, using this option https://github.com/acados/acados/blob/master/interfaces/acados_matlab_octave/acados_ocp_opts.m#L108
As of now there are no examples of its use in the matalb examples, but basically you just need to set it in the opts like any other option.

Can you add this to your examples, so there is no need to change the SQP C source code to make them converge?

After this change I'll merge your PR :)

File: examples/matlab_mex/swarming/example_ocp.m [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+time_tot = ocp.get('time_tot');
+time_lin = ocp.get('time_lin');
+time_reg = ocp.get('time_reg');
+time_qp_sol = ocp.get('time_qp_sol');
+
+fprintf('\nstatus = %d, sqp_iter = %d,  time_int = %f [ms] (time_lin = %f [ms], time_qp_sol = %f [ms], time_reg = %f [ms])\n', status, sqp_iter, time_tot*1e3, time_lin*1e3, time_qp_sol*1e3, time_reg*1e3);
+% time_ext = %f [ms], time_ext*1e3,
+
+stat = ocp.get('stat');
+if (strcmp(nlp_solver, 'sqp'))
```
Comment (FreyJo) [inline examples/matlab_mex/swarming/example_ocp.m]: for this stuff you can do `ocp.print('stat')` now :wink:

**Comment by giaf** [comment]
> Hi Enrica,
> thanks for contributing!
> Could you remove the .png file and move the example to `/examples/acados_matlab_octave/swarming/` (the directory name changed lately).

Good catch, I didn't notice the old folder name!

**Comment by EnricaSo** [comment]
Which branch should I work in? I am now on the master branch and if I ran the examples ```example_sim.m``` of the ```acados_matlab_octave``` folder, such as pendulum_on_cart_model/example_sim.m, I get 

```
Undefined function or variable 'acados_sim_model'.

Error in example_sim (line 74)
sim_model = acados_sim_model();```

**Comment by FreyJo** [comment]
> Which branch should I work in? I am now on the master branch and if I ran the examples `example_sim.m` of the `acados_matlab_octave` folder, such as pendulum_on_cart_model/example_sim.m, I get
> 
> ```
> Undefined function or variable 'acados_sim_model'.
> 
> Error in example_sim (line 74)
> sim_model = acados_sim_model();
> ```

yes, master should work fine.
It seems like something went wrong with the environment variables.
The file `acados_sim_model` is in `interfaces/acados_matlab_octave`.

If you have a env.sh file that you source it should also include the updated path `interfaces/acados_matlab_octave` which was `interfaces/matlab_mex` before.

The env.sh file should look like this
https://github.com/acados/acados/blob/master/examples/acados_matlab_octave/pendulum_on_cart_model/env.sh

**Pull request closed without merging**